my-app/src/components/login/LogInComponent.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Redirect } from "react-router-dom";
+import { login, isAuthorized } from "../../services/auth.service";
+import constantsService from "../../constants/constants.service";
+
+export default class LogInComponent extends Component {
+  state = {
+    isAuthorized: isAuthorized(), // on init
```

Comment:
Why do you put the value from function to state? Are you gonna render it somewhere